libraries/AC_Avoidance/AP_OABendyRuler.cpp

```diff
@@ -232,6 +244,159 @@ bool AP_OABendyRuler::calc_margin_from_polygon_fence(const Location &start, cons
     return true;
 }
 
+// calculate minimum distance between a path and all exclusion polygons
+// on success returns true and updates margin
+bool AP_OABendyRuler::calc_margin_from_exclusion_polygon(const Location &start, const Location &end, float &margin)
+{
+    // exit immediately if polygon fence is not enabled
```

Comment:
```suggestion
```

Reply:
ok i've removed the new comments.. there are some other identical comments in untouched code which I've left in place.